ps_facetedsearch.php

```diff
@@ -1420,7 +1420,7 @@ public function getProductByFilters(
                         ON (pa.`id_product_attribute` = pac.`id_product_attribute`)' .
                         Shop::addSqlAssociation('product_attribute', 'pa') .
                         ' JOIN `'._DB_PREFIX_.'stock_available` sa
-                        ON (sa.`id_product_attribute` = pac.`id_product_attribute` AND sa.`quantity` > 0)
+                        ON (sa.`id_product_attribute` = pac.`id_product_attribute` AND sa.`quantity` >= 0)
```

Comment:
if sa.quantity needs to be >= 0, we can remove 
```
AND sa.`quantity`
```

ping @jocel1 wdyt?

Reply:
Wonder if it could be possible to have a negative quantity here. Technically in the DB it's not an UNSIGNED so it *could* be negative

Reply:
But there are several places in the module where we are testing quantity > 0, perhaps it should be changed as well

Reply:
I think it's better to take #51 which include a switch to control this kind of behavior :)

Reply:
Yes, @PierreRambaud  I agree with you.